blackjack/index.js

```diff
@@ -39,23 +41,49 @@ function standButtonEnable() {
     standButton.disabled = false;
 }
 
-// Random value function 
-function dice() {
-    return(Math.floor(Math.random() * 13) + 1)
+// Create a 52-card deck
+function createDeck(suits, values) {
+    deck = [];
+    for (let suit of suits) {
+        for (let value of values) {
+            deck.push(value);
+        }
+    }
+
+    // Shuffle the deck
+    let counter = deck.length, temp, i;
+
+    while(counter) {
+        i = Math.floor(Math.random() * counter--);
+        temp = deck[counter];
+        deck[counter] = deck[i];
+        deck[i] = temp;
+    }
+    console.log(deck);
+    return deck;
 }
 
+// Random value function 
+// function drawACard() { // Weird function name
+//     return(Math.floor(Math.random() * 13) + 1)
+// }
+
 function gameOutcome(playerScore, dealerScore) {
     console.log(playerScore, dealerScore);
     outcomeEl.style.display = 'block';
-    if ((playerScore > 21 && dealerScore > 21) || (playerScore == dealerScore)){
+    if ((playerScore > 21 && dealerScore > 21) || (playerScore == dealerScore)){ // "10" == 10 returns true, "10" === 10 returns false
             return outcomeEl.textContent = "Draw! Play again?"
     } else if (playerScore > 21) {
             return outcomeEl.textContent = "Better luck next time!"
-    } else if (playerScore < dealerScore) {
+    } else if ((playerScore < dealerScore) && !(dealerScore > 21)) { // need to ensure that dealer haven't bust also
```

Comment:
```suggestion
    } else if ((playerScore < dealerScore) && dealerScore <= 21)) { // need to ensure that dealer haven't bust also
```
lol can just check <=